libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3535,13 +3541,8 @@ export function findCryptoCurrencyByTicker(
   return cryptocurrenciesByTicker[ticker];
 }
 
-/**
- *
- * @param {*} id
- */
-export function findCryptoCurrencyById(
-  id: string
-): CryptoCurrency | null | undefined {
+// TODO: signature should be findCryptoCurrencyById(id: CryptoCurrencyId)
+export function findCryptoCurrencyById(id: string): CryptoCurrency {
   return cryptocurrenciesById[id];
```

Comment:
I'm not sure about this change. There is a reason this is a "find" and not a "get". Use the get version instead in case you are sure about the id

Reply:
Good point.
However don't you think it can be confusing to have both available ? What is the use case for looking for a currencyId that we don't support ?

Reply:
As gre said, it's not the same. 
`find` is a search, you hope to get something but could result with nothing.
`get` *should*  result with something.

Those are heavily used pieces of software in the live, please don't change it without a good reason. You will end up breaking everything this way.

Reply:
As of now I didn't change the function and I removed the TODO but I'm still wondering what's the use case for it in the app